app/main/__init__.py

```diff
@@ -1,6 +1,20 @@
-from flask import Blueprint
+from flask import Blueprint, flash, Markup, current_app
+from flask_login import login_required, current_user
+import flask_featureflags
 
 main = Blueprint('main', __name__)
+direct_award = Blueprint('direct_award', __name__, url_prefix='/buyers/direct-award')
```

Comment:
Is this URL still up for grabs as part of the design/content makeover?  I'm not loving `direct-award` shown to users.

Reply:
_also, you said you weren't going to do that_

Reply:
Hmm I thought you meant on the page, not in the URL. It's pretty much decided, but we can discuss it tomorrow if you feel strongly about it.